.github/workflows/build-cli.yml

```diff
@@ -14,6 +14,7 @@ env:
 jobs:
   pre_job:
     runs-on: ubuntu-latest
+    if: "github.event_name != 'release' || startsWith(github.ref, 'refs/tags/hydro_cli-')"
```

Comment:
should this be `&&`?

Reply:
and then we can eliminate all conditionals

Reply:
Should still run on regular pushes too though?

Reply:
Oh true, this makes sense then!